src/pdn/test/pads_black_parrot_flipchip.ok

```diff
@@ -9,8 +9,8 @@
 [INFO ODB-0127] Reading DEF file: nangate_bsg_black_parrot/floorplan_flipchip.def
 [INFO ODB-0128] Design: soc_bsg_black_parrot
 [INFO ODB-0130]     Created 141 pins.
-[INFO ODB-0131]     Created 1489 components and 3295 component-terminals.
-[INFO ODB-0132]     Created 143 special nets and 2655 connections.
+[INFO ODB-0131]     Created 1489 components and 1523 component-terminals.
+[INFO ODB-0132]     Created 143 special nets and 883 connections.
```

Comment:
Why does this change?  All tests look similar.

Reply:
The first commit fixed some inconsistencies in the nangate45 lef for the IO pads (where some fillers had pins and other didn't), since they don't matter for the test, I removed the pins in the lef.